app/models/shop.rb

```diff
@@ -26,7 +26,9 @@ def self.last
   end
 
   def self.all
-    DB[:shops].all
+    #DB[:shops].all
+    all_records = DB[:shops].all
+    all_records.map{|x| Shop.new(x)}
```

Comment:
cool, this is the idea.
maybe we can do it in a single line, like:

```ruby
DB[:shops].all.map{ |attrs| Shop.new(attrs) }
```

Reply:
So attrs is short for attributes?